Formula/b/biosig.rb

```diff
@@ -26,12 +26,19 @@ class Biosig < Formula
   depends_on "suite-sparse"
   depends_on "tinyxml2"
 
+  on_sonoma :or_newer do
+    depends_on "libiconv"
```

Comment:
Can you explain the reason for this? A lot of `iconv` issues have been fixed so would be good to know what issues are remaining.

Reply:
Main issue looks like code doesn't handle BSD iconv correctly https://git.ista.ac.at/alois.schloegl/code/-/blob/master/biosig4c++/t210/sopen_scp_read.c?ref_type=heads#L368

```c
#if  defined(_ICONV_H) || defined (_LIBICONV_H)
```

At minimum `_ICONV_H_` is guard. Maybe just modifying that could work. Not sure which header `_ICONV_H` would be from.

Reply:
`_ICONV_H` is glibc iconv.
`_LIBICONV_H` is GNU libiconv
`_ICONV_H_` is BSD iconv.

Though those are internal defines so it probably shouldn't really be checking for things like that and should be doing configure checks instead. But as a quick patch then adding `_ICONV_H_` should work.